client/src/utils/Functions/createRefactorData.ts

```diff
@@ -0,0 +1,22 @@
+import { DisplayableData, IUnverifiedBoxes } from '../types';
+
+export const createDisplayableData = (data: IUnverifiedBoxes[]) => {
+  const displayableData = [];
+
+  for (const item of data) {
```

Comment:
Tu można by to było zrobić deklaratywnie za pomocą array.map, ale to bardziej jako ciekawostka ;)